pkg/rtc/participant.go

```diff
@@ -534,9 +534,13 @@ func (p *ParticipantImpl) handleMigrateMutedTrack() {
 	p.mutedTrackNotFired = append(p.mutedTrackNotFired, addedTracks...)
 	p.pendingTracksLock.Unlock()
 
-	for _, t := range addedTracks {
-		p.handleTrackPublished(t)
-	}
+	// launch callbacks in goroutine since they could block
+	// callbacks handle webhooks as well as db persistence
```

Comment:
Is it possible to reword this comment and the one below as these probably do not always apply in OSS code. Maybe leave them as PR comments inline rather than in code?

Reply:
I believe it does apply. participants have to be persisted to Redis